backend/src/services/certificate-authority/internal/internal-certificate-authority-fns.ts

```diff
@@ -8,6 +8,8 @@ import { getConfig } from "@app/lib/config/env";
 import { crypto } from "@app/lib/crypto/cryptography";
```

Comment:
these shouldn't have any changes as well because the `internal-` files are for the internal certificates